src/ui/link/index.tsx

```diff
@@ -49,7 +68,28 @@ const Link = forwardRef<HTMLButtonElement | HTMLAnchorElement, LinkProps>(
           ''
         : to;
 
-    const handleLinkPress = (e: any) =>
+    const triggerPrefetch = useCallback(() => {
+      // ignore if async route not ready yet
+      if (typeof to !== 'string' && !route) return;
+
+      const context =
+        typeof to !== 'string' && route
```

Comment:
Shouldn't this condition always be true with the early return above?

Reply:
Yes, but TS complains saying route could be null 😞